src/main/java/de/tum/in/www1/artemis/service/TextAssessmentService.java

```diff
@@ -112,4 +116,15 @@ public Result saveAssessment(Long resultId, List<Feedback> textAssessment, TextE
     private Double calculateTotalScore(List<Feedback> assessments) {
         return assessments.stream().mapToDouble(Feedback::getCredits).sum();
     }
+
+    /**
+     * Finds the example result for the given submission ID. The submission has to be an example submission
+     *
+     * @param submissionId The ID of the submission for which the result should be fetched
+     * @return The example result, which is linked to the submission
+     */
+    public Result getExampleResult(long submissionId) {
```

Comment:
can we move this directly into the super class and use the submissionRepository instead?